src/forms/validation-rules.js

```diff
@@ -183,9 +183,25 @@ export function validCollectedVariables(
    * It solves this issue : https://trello.com/c/bZo4vAei/397-255-questionnaire-non-r%C3%A9cup%C3%A9r%C3%A9-par-lapplication
    */
 
+  function objectCompare(object1, object2) {
```

Comment:
Why not using the isEqual of lodash we already have in our dependencies ?

Reply:
isEqual ne répond au besoin on peut tomber dans des cas de deux objets de différents taille avec des attributs vide